docs/src/gui/halui.adoc

```diff
@@ -67,9 +67,7 @@ net call-mysub     <= pyvcp.callmysub
 net call-mysub     => halui.mdi-command-02
 ----
 
-These nets connect the `halui.mdi-command-NN` pins
-provided by halui:
-
+.These nets connect the `halui.mdi-command-NN` pins provided by halui:
```

Comment:
```suggestion
These nets connect the `halui.mdi-command-NN` pins provided by halui:
```
This wording doesn't fit as a caption

Reply:
It should be "Nets connecting the `halui.mdi-command-NN` pins provided by halui."

Reply:
I mean we don't have to force using captions for small code snippets that are integrated in the text flow ...

Reply:
You use the caption when the image is not deeply embedded in the text.  I had introduced those captions for the many [underline] instructions that basically created their own header styles only to then have just that single image as a content. That image could be floating anywhere. So I prefer to have a good caption for it and focus on the main parts of the article.

I am not completely sure what the context of this block now is about. But anything that has the character of a reference   or that is an example that is not discussed in the main text, well, that to me is something that should get captions and just live somewhere happily.

What makes me shy away from captions again is that asciidoc cannot reference those items properly. It is not LaTeX after all. But here, if it does not look bad, just go for it, I suggest. That "Table xx"-prefix can be switched off if you find that disturbing - I do not like those too much, I must admit.

Reply:
If not deeply embedded in the text, the example/image/table/etc. should have a caption, I agree,  but then you should reference it in the describing text as well.

Like

>These pins can be connected like any HAL pins. A common method is to use buttons provided by virtual control panels like shown in Example 1.
>
> Example 1: Example HAL file connections
>```
>net quill-up       <= pyvcp.quillup
>net quill-up       => halui.mdi-command-00
>net reference-pos  <= pyvcp.referencepos
>net reference-pos  => halui.mdi-command-01
>net call-mysub     <= pyvcp.callmysub
>net call-mysub     => halui.mdi-command-02
>```

But this is not possible with our Asciidoc version. Up from 1.5.6 it should be possible maybe (https://github.com/asciidoctor/asciidoctor/issues/858)

Currently we can have it only like this:

![image](https://user-images.githubusercontent.com/67957916/188594025-dfd5f350-b30d-48e9-81b5-4e2422ab64cb.png)


Otherwise I would prefer:

>These pins can be connected like any HAL pins. A common method is to use buttons provided by virtual control panels like:
>
>```
>net quill-up       <= pyvcp.quillup
>net quill-up       => halui.mdi-command-00
>net reference-pos  <= pyvcp.referencepos
>net reference-pos  => halui.mdi-command-01
>net call-mysub     <= pyvcp.callmysub
>net call-mysub     => halui.mdi-command-02
>```

Reply:
I think we are in sync.

How to proceed? Yet another build? I would otherwise just ask you to amend to your liking at commit.

Reply:
Push, I mean.

Reply:
Commit and Push.

Reply:
So do you agree to my workaround?

Reply:
What about this?
![image](https://user-images.githubusercontent.com/67957916/188600591-a69014f5-b487-4038-b84a-77fde21d8a64.png)